docs/process/issue-triage.md

```diff
@@ -98,11 +98,14 @@ issues from time to time that isn't and won't be covered here.
 
 ## More Information Needed
 
-Periodically we should be doing a sweep of issues that are open and labeled
-`more-information-needed`. If the original poster has not responded within
-two weeks after the last question by an official maintainer, close the issue
-with a message saying that you're closing the issue due to inactivity but are
-happy to re-open if they can provide more details.
+If we cannot understand or reproduce the issue with the information given, then we label it `more-information-needed`.
+
+Periodically, we should be doing a sweep of issues that are open and labeled
```

Comment:
what do we mean by periodically (i know this was there before, i'm just opening this up to a larger question) -- is that once during a FRR shift (aka 1x/week) or every day? what's the order of magnitude?

Reply:
My assumption is we do this during FRR, but that still doesn't state how often. I don't think we need to say how often though since Probot will catch it after 14 days.

Reply:
i'd imagine we should be doing a sweep once a week due to bullet point #2

Reply:
Went ahead and specified how often this should be done.